dropwizard-jetty/src/main/java/io/dropwizard/jetty/LocalIpFilter.java

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
I only added the license header here because I basically copied this code out of Curator.  However, since both Curator and Dropwizard are under the Apache 2.0 license, maybe this is unnecessary?